build.gradle

```diff
@@ -32,6 +32,10 @@ buildscript {
         // Needed for caching reflected data during builds
         classpath 'org.reflections:reflections:0.9.10'
         classpath 'dom4j:dom4j:1.6.1'
+
+        // plugin for compilation Protobuf files
+        // The latest version is 0.8.1, but it requires Gradle 2.12
```

Comment:
Updating to 2.12 or even 2.14 probably wouldn't be terrible. Gradle 3 I think might come with some other required changes or we'd need to test for quirks more thoroughly.